LISISO/commonfunctions.sh

```diff
@@ -315,3 +315,62 @@ function checkrpms()
 		echo "RPM's are missing"
 	fi
 }
+
+function CheckLibModulePartition()
```

Comment:
CheckLibModulePartition and CheckBootPartition can be optmized to have single function

Reply:
Optimized to one function